package.json

```diff
@@ -43,6 +43,7 @@
     "clear": "^0.0.1",
     "cliui": "^3.2.0",
     "clone": "^2.1.1",
+    "core-js": "2.5.3",
```

Comment:
I think we should be good with default semver range `^2.5.3` :)

Reply:
I'll add the `^`!